include/seqan3/core/detail/customisation_point.hpp

```diff
@@ -54,3 +56,109 @@ static constexpr decltype(auto) impl(seqan3::detail::priority_tag<PRIO>,
 {                                                                                                                    \
     return TERM;                                                                                                     \
 }
+#endif // SEQAN3_DEPRECATED_310
+
+#if SEQAN3_DOXYGEN_ONLY(1)0
+//!\brief A macro helper for #SEQAN3_CPO_OVERLOAD.
```

Comment:
```suggestion
/*!\brief A macro helper for #SEQAN3_CPO_OVERLOAD.
 * \details
 * Please note that to allow the macro to end with a semi-colon, we need some expression that ends with a semi-colon.
 * We chose `static_assert(true)` for that reason.
 */
```

Reply:
```suggestion
/*!\brief A macro helper for #SEQAN3_CPO_OVERLOAD.
 * \details
 * Please note that in order to allow a semicolon at the end when using this macro, i.e.
 *
 * ```cpp
 *  SEQAN3_CPO_OVERLOAD_BODY
 *  (
 *      ...
 *  );
 * ```
 *
 * we need some expression at the end of the macro that can have a semicolon appended.
 * We chose `static_assert(true)` for that reason.
 */
```